spec/abci/abci++_basic_concepts.md

```diff
@@ -311,9 +314,11 @@ message Event {
 }
 ```
 
-The attributes of an `Event` consist of a `key`, a `value`, and an `index` flag. The
-index flag notifies the CometBFT indexer to index the attribute. The value of
-the `index` flag is non-deterministic and may vary across different nodes in the network.
+The attributes of an `Event` consist of a `key`, a `value`, and an `index`
+flag. The index flag notifies the CometBFT indexer to index the attribute.
+
+The `type` and `attributes` fields are non-deterministic and may vary across
+different nodes in the network.
```

Comment:
What about the `index` flag, it should keep being listed as non-deterministic too?

Reply:
`attributes` includes `index`